pulp_rpm/app/serializers/repository.py

```diff
@@ -318,6 +342,24 @@ class RpmRepositorySyncURLSerializer(RepositorySyncURLSerializer):
         help_text=_("Whether or not to optimize sync."), required=False, default=True
     )
 
+    def validate(self, data):
+        """
+        Validate sync parameters.
+        """
+        data = super().validate(data)
+        mirror = data.get("mirror", None)
+        sync_policy = data.get("sync_policy", None)
+
+        if mirror and sync_policy:
```

Comment:
if 'mirror' in data and 'sync_policy' in data....

otherwise
```
$ http :24817/pulp/api/v3/repositories/rpm/rpm/46335757-c221-4539-874b-6e21bcce6674/sync/ remote=/pulp/api/v3/remotes/rpm/rpm/3a95a54f-31cb-40ff-a7fa-33c38ed8085d/ sync_policy=mirror_complete mirror=False
HTTP/1.1 202 Accepted
Access-Control-Expose-Headers: Correlation-ID
Allow: POST, OPTIONS
Connection: close
Content-Length: 67
Content-Type: application/json
Correlation-ID: 0313a5fad5314321b0ce0aac60497ef6
Date: Fri, 15 Oct 2021 10:36:53 GMT
Referrer-Policy: same-origin
Server: gunicorn
Vary: Accept, Cookie
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{
    "task": "/pulp/api/v3/tasks/f5511ee9-3efa-4d22-a5db-9638c2412c3a/"
}


```